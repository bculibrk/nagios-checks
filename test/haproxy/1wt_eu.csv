# pxname,svname,qcur,qmax,scur,smax,slim,stot,bin,bout,dreq,dresp,ereq,econ,eresp,wretr,wredis,status,weight,act,bck,chkfail,chkdown,lastchg,downtime,qlimit,pid,iid,sid,throttle,lbtot,tracked,type,rate,rate_lim,rate_max,check_status,check_code,check_duration,hrsp_1xx,hrsp_2xx,hrsp_3xx,hrsp_4xx,hrsp_5xx,hrsp_other,hanafail,req_rate,req_rate_max,req_tot,cli_abrt,srv_abrt,comp_in,comp_out,comp_byp,comp_rsp,lastsess,last_chk,last_agt,qtime,ctime,rtime,ttime,agent_status,agent_code,agent_duration,check_desc,agent_desc,check_rise,check_fall,check_health,agent_rise,agent_fall,agent_health,addr,cookie,mode,algo,conn_rate,conn_rate_max,conn_tot,intercepted,dcon,dses,
http-in,FRONTEND,,,5,25,100,3161987,609712281,6469035959,2232237,0,10310,,,,,OPEN,,,,,,,,,1,2,0,,,,0,3,0,89,,,,0,876125,59515,55972,244,2230104,,3,89,3211133,,,0,0,0,0,,,,,,,,,,,,,,,,,,,,,http,,3,89,3181215,354602,0,0,
http-in,IPv4-direct,,,1,25,100,2291028,367022152,1064567588,2219307,0,17,,,,,OPEN,,,,,,,,,1,2,1,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,http,,,,,,0,0,
http-in,IPv4-cached,,,0,14,100,812576,223558890,4917830438,6637,0,1,,,,,OPEN,,,,,,,,,1,2,2,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,http,,,,,,0,0,
http-in,IPv6-direct,,,0,12,100,45357,15381777,292639605,2665,0,10292,,,,,OPEN,,,,,,,,,1,2,3,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,http,,,,,,0,0,
http-in,local,,,0,0,100,0,0,0,0,0,0,,,,,OPEN,,,,,,,,,1,2,4,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,http,,,,,,0,0,
http-in,local-https,,,0,5,100,32254,3749462,193998328,3628,0,0,,,,,OPEN,,,,,,,,,1,2,5,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,http,,,,,,0,0,
www,www,0,0,0,14,20,459085,169826821,3914623823,,0,,10,15,46,2,DOWN,1,1,0,26,7,885894,895155,,1,3,1,,459078,,2,0,,68,L7STS,404,2,0,410203,11256,37455,1,0,,,,,42593,2,,,,,885895,Not Found,,0,3,37,513,,,,Layer7 wrong status,,2,3,0,,,,,,http,,,,,,,,
www,bck,0,1,4,10,10,97391,38690561,1971792825,,0,,2,0,2,0,UP,1,0,1,9,2,4028243,1154,,1,3,2,,97394,,2,0,,59,L7OK,200,2,0,90656,1418,5292,0,0,,,,,19650,2,,,,,20,OK,,0,1,17,1044,,,,Layer7 check passed,,2,5,6,,,,,,http,,,,,,,,
www,BACKEND,0,0,0,21,100,558269,208605416,5886751764,1716,0,,71,15,48,2,UP,1,0,1,,2,4028243,1154,,1,3,0,,556472,,1,0,,68,,,,0,500859,12674,44473,220,43,,,,558269,62274,4,0,0,0,0,20,,,0,1,17,1044,,,,,,,,,,,,,,http,,,,,,,,
git,www,0,0,0,2,2,1464,580155,27040897,,0,,0,0,0,0,DOWN,1,1,0,21,7,885894,895119,,1,4,1,,1334,,2,0,,2,L7STS,404,1,0,1210,0,254,0,0,,,,,15,0,,,,,886879,Not Found,,421,1,1089,2042,,,,Layer7 wrong status,,2,3,0,,,,,,http,,,,,,,,
git,bck,0,7,0,2,2,356,129900,11454350,,0,,0,0,0,0,UP,1,0,1,9,2,4028243,1160,,1,4,2,,356,,2,0,,2,L7OK,200,2,0,325,0,31,0,0,,,,,27,0,,,,,3209,OK,,52,1,537,1334,,,,Layer7 check passed,,2,3,4,,,,,,http,,,,,,,,
git,BACKEND,0,10,0,13,2,1820,750189,38495247,0,0,,0,0,0,0,UP,1,0,1,,2,4028243,1160,,1,4,0,,1690,,1,0,,4,,,,0,1535,0,285,0,0,,,,1820,42,0,0,0,0,0,3209,,,262,1,1080,2352,,,,,,,,,,,,,,http,,,,,,,,
demo,BACKEND,0,0,1,10,20,19867,7087787,483986641,40,0,,24,0,0,0,UP,0,0,0,,0,5344748,,,1,17,0,,0,,1,1,,8,,,,0,19129,243,0,24,470,,,,19866,86,158,0,0,0,0,0,,,0,0,0,2065,,,,,,,,,,,,,,http,,,,,,,,
