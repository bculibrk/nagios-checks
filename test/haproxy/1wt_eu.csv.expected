HAPROXY CRIT: www BACKEND (act) DOWN	sess=0/100(0%) smax=21
demo BACKEND (act) UP	sess=1/20(5%) smax=10
git BACKEND (act) UP	sess=0/2(0%) smax=13
git bck (bck) UP	sess=0/2(0%) smax=2
git www (act) UP	sess=0/2(0%) smax=2
http-in FRONTEND (act) OPEN	sess=5/100(5%) smax=25
http-in IPv4-cached (act) OPEN	sess=0/100(0%) smax=14
http-in IPv4-direct (act) OPEN	sess=1/100(1%) smax=25
http-in IPv6-direct (act) OPEN	sess=0/100(0%) smax=12
http-in local (act) OPEN	sess=0/100(0%) smax=0
http-in local-https (act) OPEN	sess=0/100(0%) smax=5
www BACKEND (act) DOWN	sess=0/100(0%) smax=21
www bck (bck) UP	sess=4/10(40%) smax=10
www www (act) DOWN	sess=0/20(0%) smax=14
